modules/KIWILinuxRC.sh

```diff
@@ -1324,8 +1324,10 @@ EOF
     # install grub2 in BIOS mode
     #--------------------------------------
     if [ ! -z "$kiwi_OfwGrub" ];then
+        local prepdev=$imagePrepDevice
+        test -z $prepdev && prepdev=$(ddn $imageDiskDevice $kiwi_OfwGrub)
```

Comment:
Well, it is used in KIWILinuxRC.sh

the code was added by me and I'm sure I've tested it.